.env.template

```diff
@@ -1,23 +1,23 @@
 HOST='0.0.0.0'
 PORT=3000
-REACT_APP_WS_URL=ws://wallet-backend-server:8002
-REACT_APP_WALLET_BACKEND_URL=http://wallet-backend-server:8002
-REACT_APP_LOGIN_WITH_PASSWORD=false
-REACT_APP_FIREBASE_ENABLED=false
-REACT_APP_FIREBASE_VAPIDKEY=<Your_Vapid_Key>
-REACT_APP_FIREBASE_API_KEY=<Your_Firebase_API_Key>
-REACT_APP_FIREBASE_AUTH_DOMAIN=<Your_Firebase_Auth_Domain>
-REACT_APP_FIREBASE_PROJECT_ID=<Your_Firebase_Project_ID>
-REACT_APP_FIREBASE_STORAGE_BUCKET=<Your_Firebase_Storage_Bucket>
-REACT_APP_FIREBASE_MESSAGING_SENDER_ID=<Your_Firebase_Messaging_Sender_ID>
-REACT_APP_FIREBASE_APP_ID=<Your_Firebase_App_ID>
-REACT_APP_FIREBASE_MEASUREMENT_ID=<Your_Firebase_Measurement_ID>
-REACT_APP_DID_KEY_VERSION=jwk_jcs-pub
-REACT_APP_VERSION=$npm_package_version
+VITE_WS_URL=ws://wallet-backend-server:8002
+VITE_WALLET_BACKEND_URL=http://wallet-backend-server:8002
+VITE_LOGIN_WITH_PASSWORD=false
+VITE_FIREBASE_ENABLED=false
+VITE_FIREBASE_VAPIDKEY=<Your_Vapid_Key>
+VITE_FIREBASE_API_KEY=<Your_Firebase_API_Key>
+VITE_FIREBASE_AUTH_DOMAIN=<Your_Firebase_Auth_Domain>
+VITE_FIREBASE_PROJECT_ID=<Your_Firebase_Project_ID>
+VITE_FIREBASE_STORAGE_BUCKET=<Your_Firebase_Storage_Bucket>
+VITE_FIREBASE_MESSAGING_SENDER_ID=<Your_Firebase_Messaging_Sender_ID>
+VITE_FIREBASE_APP_ID=<Your_Firebase_App_ID>
+VITE_FIREBASE_MEASUREMENT_ID=<Your_Firebase_Measurement_ID>
+VITE_DID_KEY_VERSION=jwk_jcs-pub
+VITE_VERSION=$npm_package_version
```

Comment:
This should rather be named `APP_VERSION` or maybe `VITE_APP_VERSION` to not look like it's the version of Vite rather than the version of the app.